### ISSUE TITLE
Use static User Agent

### ISSUE THREAD

**Issue by TomBrien** [TomBrien/cardiffwaste-ha#28]
Probably best just to use a static up to date user agent

**Comment by inverse** [comment]
What's the rational for the randomisation in the first instance?

Were they blocking requests when using the default httpx one. e.g. `python-httpx/0.28.1`?


p.s. should this issue live on the lib repo (https://github.com/TomBrien/cardiffwastepy)

**Comment by TomBrien** [comment]
Yeah it blocks the default but also repeated calls from the same, which is a pain in debugging
